Find route in transport network

shubham7109 opened the pull request:
## Description
PR to add a new Kotlin sample _"Find route in transport network"_ in `Routing and Logistics` category.

## Links and Data
Sample Epic: `runtime/kotlin/issues/1413`

hud10837 commented on find-route-in-transport-network/src/main/java/com/esri/arcgismaps/sample/findrouteintransportnetwork/DownloadActivity.kt:
```diff
@@ -0,0 +1,22 @@
+
+        )
+    }
+}
```
Missing newline

hud10837 reviewed (approved):
LGTM and works great. I like the new UI as well.

prupani-7 reviewed (commented):
@shubham7109 Overall, the sample looks good, and works smoothly!
Left a few minor comments.

I have a question: any reason why we are not showing a boundary symbol around the envelope?
If 2 points are tapped on the map and one of those is outside the envelope, then the route looks disconnected like this:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/44178002/214977054-e6004f05-795d-4bec-a469-030df823b9f1.png">
I believe this is expected. Although a bit confusing to the user if we don't show the boundary outlining the envelope. I see the old kotlin sample having that.

What do you think about this?

shubham7109 commented on find-route-in-transport-network/src/main/AndroidManifest.xml:
```diff
@@ -0,0 +1,32 @@
+            android:label="@string/app_name" />
+    </application>
+
+</manifest>
```
I don't think Android studio creates a new line here by default anymore. But I'll change it for consistency.

shubham7109 commented on find-route-in-transport-network/src/main/java/com/esri/arcgismaps/sample/findrouteintransportnetwork/MainActivity.kt:
```diff
@@ -0,0 +1,313 @@
+/* Copyright 2022 Esri
```
This is a good reminder for me to automate this is the new module script.

shubham7109 commented:
@prupani-7 Thanks for the review, I've added the feedback. About your comment on the tapped location, I've added in the message: "Tapped location is outside the transport network". Since this version of the sample just uses the envelope as the viewpoint it should cover most of the routes on the screen. Also, I tried out the suggestion and I don't think we need a border on a map unless it's necessary just for sample simplicity. Screenshot: 

![screenshot](https://user-images.githubusercontent.com/14892474/214984093-db26a429-c4f9-4cd4-ab67-588dd1a91028.png)

prupani-7 reviewed (approved):
Thanks! LGTM
